sopel/tools/calculation.py

```diff
@@ -223,8 +238,8 @@ def __init__(self):
             unary_ops=self.__unary_ops
         )
 
-    def __call__(self, expression_str):
-        result = ExpressionEvaluator.__call__(self, expression_str)
+    def __call__(self, expression_str: str, timeout: Union[float, int] = 5.0):
```

Comment:
Same nit as above, `float` allows `int`

```suggestion
    def __call__(self, expression_str: str, timeout: float = 5.0):
```